src/spoof.ts

```diff
@@ -87,45 +75,42 @@ export const createCursor = (page: Page, start: Vector = origin) => {
       const elem = await page.$(selector);
       if (!elem) {
         throw new Error(
-          `Could not find element with selector "${selector}", make sure you're waiting for the elements with "puppeteer.waitForSelector"`
+          `Could not find element with selector "${selector}", make sure you're waiting for the elements with "puppeteer.waitForSelector"`,
         );
       }
+      // Make sure the object is in view
+      let objectId;
+      if ((objectId = (elem as any)._remoteObject.objectId)) {
+        await (page as any)._client.send('DOM.scrollIntoViewIfNeeded', {
+          objectId,
```

Comment:
Looks like [scrollintoViewIfNeeded](https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoViewIfNeeded) is deprecated. Can you also change this up so it doesn't use private properties on puppeteer?

Reply:
You're looking at the wrong documentation, that's the DOM scrollIntoViewIfNeeded call. This PR is using the CDP call instead: https://chromedevtools.github.io/devtools-protocol/tot/DOM/#method-scrollIntoViewIfNeeded. That's the only way to do this without detection.

Reply:
The only other way to call CDP methods directly is by creating a new CDP session: https://pptr.dev/#?product=Puppeteer&version=v5.4.1&show=api-class-cdpsession

That's more "type safe" but also more overhead as it creates an extra session.

Reply:
I added eslint config, `yarn lint` support and a `getRandomPagePoint()` function as well so it doesn't always start at 0,0

Reply:
Also switched to puppeteer-core for smaller dependencies and updated the types